### ISSUE TITLE
🗓 Create single event from week calendar view

### ISSUE THREAD

**Issue by richardolsson** [zetkin/app.zetkin.org#1242]
## Description
A user should be able to add new events by dragging the desired timespan in the week calendar view

## Relevant Job Stories
None

## Prerequisites
Don't start this until #1235 has been completed.

## Requirements
- [x] Clicking and dragging in a calendar day should "draw" a rectangle that fills the width of the column
- [x] When the mouse is released, open a small popper with one option, "Single event"
- [x] Clicking "Single event" should create a new event (with no location and no activity) and navigate to that event

## Possible implementations
None

## Design specifications
![image](https://user-images.githubusercontent.com/550212/233618115-05c5b307-8651-4942-b140-392ce7a98a0c.png)

## Open questions
None

**Comment by richardolsson** [comment]
Resolved by #1278 and merged into #1257
